fio/lib/README.md

```diff
@@ -0,0 +1,54 @@
+# FIOWallet Class
+
+The FIOWallet class provides functionalities for interacting with the FIO blockchain, including mapping a public address to a FIO address.
+
+## Installation
+
+To use the FIOWallet class in your project, you can install it via npm:
+
+```bash
+npm install fio-wallet
+
+yarn add fio-wallet
+```
+
+## Usage
+
+### Creating a FIOWallet Instance
+
+- First, import the FIOWallet class:
+
+```js
+import { FIOWallet } from "fiowallet";
+```
+
+- Then, create an instance of FIOWallet by providing the required parameters:
+
+```js
+const appId = "your_app_id";
+const email = "your_email@example.com";
+const wallet = new FIOWallet(appId, email);
+```
+
+## Mapping a Public Address to a FIO Address
+
+### To map a public address to a FIO address, call the mapHandle method on the FIOWallet instance:
+
+```js
+const publicAddress = "your_public_address";
+wallet.mapHandle({
+  publicAddress: publicAddress,
+  chainCode: "ETH",
+  tokenCode: "ETH",
+});
+```
+
+Make sure to replace 'your_public_address' with the actual public address you want to map, and provide the necessary chain code and token code.
```

Comment:
Add a link to how to find out chainCode and tokenCode

Reply:
Done